### ISSUE TITLE
Check whatever Instagram app for Android is using OSM maps without a proper attribution

### ISSUE THREAD

**Issue by matkoniecz** [matkoniecz/illegal-use-of-OpenStreetMap#2]
Sadly, Instagram appears to block popular throwaway mail services and it is impossible to view content without registering.

Has anybody access to an Instagram account and can check this?

**Comment by ForgottenHero** [comment]
Instagram DOES use OSM.
When viewing a location where a photo was tagged at, it displays a map at the top of the screen.
There is a small ℹ (information) button in the corner which I personally found very hard to press.
Pressing on the map itself opens in whichever map app the user's mobile device has installed.

Press on a user's photo location:
![Insta - East Point Lighthouse](https://user-images.githubusercontent.com/4233945/101792664-b11fb600-3b50-11eb-8784-5610e27281b0.jpg)

Press on "More Information":
![Insta - East Point Lighthouse - More](https://user-images.githubusercontent.com/4233945/101792713-c1d02c00-3b50-11eb-82eb-7f3d4831c81a.jpg)

Press on the small info button in the corner:
![Insta - East Point Lighthouse - Info](https://user-images.githubusercontent.com/4233945/101792758-cf85b180-3b50-11eb-808c-67a7d2955f75.jpg)

This attribution page links to the following site: [https://m.facebook.com/maps/attribution_terms/](https://m.facebook.com/maps/attribution_terms/).

In summary:

* [x] are there any maps or geographic data in this app?
* [x] is it from OSM? (or there is nothing that could be from OSM)
* [ ] is it clearly credited to OSM? (or there are no OSM data used)
* [x] can you take a screenshot and share it here? (or confirm that this app seems to not be using OSM data without attribution?)

**Comment by matkoniecz** [comment]
Thanks for checking! I will add it, hopefully soon.

**Comment by Firefishy** [comment]
Reports that Instagram have updated their map again: https://techcrunch.com/2022/07/19/instagram-new-searchable-map-experience/

Still OpenStreetMap?

**Comment by ForgottenHero** [comment]
Yes, Instagram is still using OSM.

1. From a post, tap on the location.
2. This presents the map.
3. There is an info icon in the bottom left of the map (previously in the bottom right corner).
4. Tap the info icon.
5. You are presented with the following text (links in brackets added by me):

> Map information
> Report a problem with the map (https://www.openstreetmap.org/fixthemap)
> Map data legal notices (https://m.facebook.com/maps/attribution_terms/)
> © OpenStreetMap (https://www.openstreetmap.org/copyright)

![image](https://user-images.githubusercontent.com/4233945/222147980-13543dc6-7dbd-42b5-881f-7232228c1b0a.png)

Here's a recording if it helps at all: https://youtube.com/shorts/UTa0aFIXdT0

I suppose what I've just found on desktop would probably be another issue altogether but there _appears_ to be no attribution when viewed on desktop _if using dark mode_.
As an example I found a post with Brisbane as the location. It took me here: https://www.instagram.com/explore/locations/213570382/brisbane-queensland-australia/
This is what I saw:
![image](https://user-images.githubusercontent.com/4233945/222147755-c35e8886-4ab5-4c95-839c-5a9c201405b3.png)

You cannot interact with the map at all (no panning, zooming, or clicking).
While in dark mode (built-in dark mode offered by Instagram, not a 3rd party extension) I could barely make out the info button.
When I clicked it, the text isn't even visible.
Poor UX decisions aside, it's the same menu as on mobile.
![image](https://user-images.githubusercontent.com/4233945/222148566-862d6450-761c-4af2-b582-a6e0f0163fd9.png)

Swapping to light mode resolves the visibility issue.
![image](https://user-images.githubusercontent.com/4233945/222148841-713a18f8-6294-4096-b146-9e7d4d3842af.png)
